src/js/sync-to-drive.js

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env node
+/* eslint-disable node/shebang */
+
+const {execSync, spawn} = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const myPid = process.pid;
```

Comment:
Move line down to other constants, so we can keep imports clean

Reply:
Yes, thank you. I'll add in to a PR